src/shapes/Polyline.ts

```diff
@@ -146,20 +144,26 @@ export class Polyline<
       offsetX - offsetY * Math.tan(degreesToRadians(this.skewX));
     const pathOffsetY =
       offsetY - pathOffsetX * Math.tan(degreesToRadians(this.skewY));
-    // TODO: remove next line
-    const legacyCorrection =
-      !this.fromSVG && !this.exactBoundingBox ? this.strokeWidth / 2 : 0;
     return {
       ...bbox,
-      left: bbox.left - legacyCorrection,
-      top: bbox.top - legacyCorrection,
       pathOffset: new Point(pathOffsetX, pathOffsetY),
       strokeOffset: new Point(bboxNoStroke.left, bboxNoStroke.top).subtract(
```

Comment:
i can't find what `strokeOffset` is used for, but i m not touching it.
Is always set but never read.

Reply:
I believe I exposed it because fabric positions an object with stroke taken into account, so I thought of cases which a dev would want to negate that.
Should be removed in #8767